pipeline_tools/shared/submission/create_file_descriptor.py

```diff
@@ -27,9 +29,11 @@ def build_file_descriptor(
     """
 
     SCHEMA_TYPE = 'file_descriptor'
-    file_id = get_uuid5(get_uuid5(sha256))
+    relative_location = get_relative_file_location(file_path)
     file_version = convert_datetime(creation_time)
 
+    file_id = get_uuid5(get_uuid5(str(project_id) + relative_location))
```

Comment:
So `file_id` is the UUID5 of "UUID5 of the `str(project_id) + relative_location` as the SHA"?

Reply:
```suggestion
    file_id = get_uuid5(get_uuid5(f"{project_id}{relative_location}"))
```